reference/forms/types/options/group_by.rst.inc

```diff
@@ -26,7 +26,7 @@ Take the following example::
             '1 month' => new \DateTime('+1 month'),
         ),
         'choices_as_values' => true,
-        'group_by' => function($value, $key, $index) {
+        'group_by' => function($vategory, $key, $value) {
             if ($value <= new \DateTime('+3 days')) {
```

Comment:
Here the comparison should be against the first argument `$choiceValue`. I will check that for 2.8. But it's definitely true for 4.0. The first argument of the callback is the model data. The third `$value` is just a string from request.